src/householder/householder.service.spec.ts

```diff
@@ -1,20 +1,74 @@
 import type { TestingModule } from '@nestjs/testing';
 import { Test } from '@nestjs/testing';
+import type { MockProxy } from 'jest-mock-extended';
+import { mockDeep } from 'jest-mock-extended';
 
+import { PrismaService } from '../prisma/prisma.service';
 import { HouseholderService } from './householder.service';
+import {
+  createHouseholderInput,
+  householderStub,
+} from './stubs/householder.stub';
 
 describe('HouseholderService', () => {
   let service: HouseholderService;
+  let prisma: MockProxy<PrismaService>;
 
   beforeEach(async () => {
     const module: TestingModule = await Test.createTestingModule({
-      providers: [HouseholderService],
+      providers: [
+        HouseholderService,
+        {
+          provide: PrismaService,
+          useFactory: () => mockDeep<PrismaService>(),
+        },
+      ],
     }).compile();
 
     service = module.get<HouseholderService>(HouseholderService);
+    prisma = module.get(PrismaService);
   });
 
   it('should be defined', () => {
     expect(service).toBeDefined();
   });
+
+  describe('CreateHouseholder', () => {
+    it('should be defined', () => {
+      // eslint-disable-next-line @typescript-eslint/unbound-method
+      expect(service.create).toBeDefined();
+    });
+
+    it('should called with', async () => {
+      const test = jest.spyOn(service, 'create');
+      await service.create(createHouseholderInput);
+
+      expect(test).toHaveBeenCalledWith(createHouseholderInput);
+    });
+
+    it('should fail: throw categorized error', async () => {
+      const errorMessage = 'householder can not created';
+
+      jest
+        .spyOn(prisma.householder, 'create')
+        .mockRejectedValue(new Error(errorMessage));
+
+      await expect(service.create(createHouseholderInput)).rejects.toThrow(
+        errorMessage,
+      );
```

Comment:
It could be better if you define a new variable and assign `service.create(... `to it for more readability